Projects/binarysearch.cpp

```diff
@@ -0,0 +1,42 @@
+#include <iostream>
```

Comment:
Add comments!

Reply:
You are supposed to make changes in Files @visakh2. i will close It After 8.00 pm! 
Please Do it!